managed/ui/src/components/universes/ToggleBackupState/ToggleBackupState.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { YBModal } from '../../common/forms/fields';
+
+/**
+ * recieves current universe details with modal controls.
+ * @param {visible, onHide, universe, updateBackupState} props 
+ * @returns - confirmation modal to toggle backup state
+ */
+export const ToggleBackupState = (props) => {
+  const {
+    visible,
+    onHide,
+    universe: { universeConfig },
+    updateBackupState
+  } = props;
+
+  let title, modalTitle = null;
```

Comment:
I'm not sure of these naming convention `title and modalTitle`. Can we do something else like `modalHeading` which will have `Enable/Disable Backups` and `title` which will carry the main title like `Enable/Disable Backup for "universe name"`. And no need of `:` after `for`.